ci/build.sh

```diff
@@ -4,4 +4,33 @@ set -e
 # You can run it from any directory.
 PROJECT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/.."
 
+source "$PROJECT_DIR/ci/git_branch.sh"
+
 "$PROJECT_DIR"/gradlew --no-daemon --info --stacktrace clean build
+
+BINTRAY_USER="$BINTRAY_USER"
+BINTRAY_API_KEY="$BINTRAY_API_KEY"
+
+GIT_BRANCH=`getCurrentGitBranch`
+
+function checkIfEnvVarIsNotEmpty() {
+    local env_var_name="$1"
+
+    if [ -z "env_var_name" ]; then
```

Comment:
Not a bash expert, but shouldn't this be `"$env_var_name"`?

Reply:
facepalm